notebooks/classification/tutorial_multilabel_classification_precision.ipynb

```diff
@@ -0,0 +1,260 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we use LTT to create prediction sets such that\n",
+    "precision score is controlled. The LTT method isimplemented in MAPIE,\n",
```

Comment:
```suggestion
    "precision score is controlled. The LTT method is implemented in MAPIE,\n",
```

Reply:
@SZiane, can you check all the cells in the notebook to make sure there are no typos?

Reply:
Done